HEFSConverter/DssEnsembleWriter.cs

```diff
@@ -12,7 +13,7 @@ namespace HEFSConverter
 {
   public class DssEnsembleWriter
   {
-    public static TimeSpan Write(string dssFileName, ITimeSeriesOfEnsembleLocations watersheds, bool saveAsFloat, int version)
+    public static TimeSpan Write(string dssFileName, TimeSeriesOfEnsembleLocations watersheds, bool saveAsFloat, int version)
```

Comment:
who needs inversion of control anyway.

Reply:
We should also drop the TimeSpan.  Higher level code does the timing now.